packages/react/src/components/carbon-components-react/UIShell/SideNavMenu.js

```diff
@@ -104,10 +104,16 @@ export class SideNavMenu extends React.Component {
     };
   }
 
-  handleToggleExpand = () => {
+  handleToggleExpand = event => {
     this.setState(state => ({ isExpanded: !state.isExpanded }));
   };
 
+  handleKeyToggleExpand = event => {
+    if (event.charCode === 13 || event.charCode === 32) {
```

Comment:
Just for the sake of a modern way here...:

```suggestion
    if (event.key === 'Enter' || event.key === ' ') {
```

Reply:
@asudoh Haha I actually had it that way at first, but changed it because it seemed it a bit too semantic. I can change that np 👍